lib/xml_builder.ex

```diff
@@ -180,44 +181,44 @@ defmodule XmlBuilder do
       iex> XmlBuilder.generate({:person, %{id: 1}, "Steve Jobs"})
       "<person id=\\\"1\\\">Steve Jobs</person>"
   """
-  def generate(any),
-    do: format(any, 0) |> IO.chardata_to_string
+  def generate(any, options \\ []),
```

Comment:
Should we add an example to the doctype for `indent: false`?

Reply:
sure thing.